lambda_handlers/formatters/tests/test_json_formatter.py

```diff
@@ -0,0 +1,60 @@
+import json
+from typing import Dict, Union
+
+import pytest
+
+from lambda_handlers.formatters import InputFormat, OutputFormat
+
+
+@pytest.fixture
+def data_sample() -> Dict[str, Union[str, int, bool]]:
+    return {
+        'name': 'Peter',
+        'score': 100,
+        'valid': True,
+    }
+
+
+class TestInputFormatJSON:
+
+    def test_valid(self, data_sample):
+        assert data_sample == InputFormat.json(json.dumps(data_sample))
+
+    def test_empty_content(self):
+        assert {} == InputFormat.json('{}')
+
+    def test_random_string(self):
+        with pytest.raises(json.decoder.JSONDecodeError, match='Expecting value:'):
```

Comment:
any reason to leave the match parameter truncated?

Reply:
Added the whole messages